tools/gn

```diff
@@ -147,15 +147,21 @@ def to_gn_args(args):
     gn_args['skia_use_wuffs'] = True
     gn_args['skia_use_expat'] = args.target_os == 'android'
     gn_args['skia_use_fontconfig'] = args.enable_fontconfig
-    gn_args['flutter_use_fontconfig'] = args.enable_fontconfig
-    gn_args['flutter_enable_skshaper'] = args.enable_skshaper
+    if args.target_os != 'wasm':
+      gn_args['flutter_use_fontconfig'] = args.enable_fontconfig
+      gn_args['flutter_enable_skshaper'] = args.enable_skshaper
```

Comment:
Why? Don't we want to use SkShaper if wasm?

Reply:
`flutter_enable_skshaper` is unused if you don't build Flutter, so setting it will cause `gn` to fail